hyperdome_client/hyperdome_client.py

```diff
@@ -331,13 +341,15 @@ def counselor_got_guest(guest_key: str):
                     self.session, self.server, self.uid
                 )
                 self.get_messages_task.setAutoDelete(False)
+                try:
+                    self.get_messages_task.signals.disconnect()
+                except TypeError:
+                    pass
```

Comment:
Same as 324

Reply:
I'll add a comment on why this is happening